tests/ctst/world/Zenko.ts

```diff
@@ -134,47 +193,48 @@ export default class Zenko extends World {
      * @returns {undefined}
      */
     async setupEntity(entityType: string): Promise<void> {
-        const keycloakPassword = this.parameters.KeycloakPassword || '123';
-        const savedParameters = JSON.parse(JSON.stringify(this.cliOptions));
+        const usedParameters = this.parameters;
```

Comment:
the name `usedParameters` is not very appropriate, maybe let's call it `defaultParameters`